### ISSUE TITLE
Create matrices by supplying vectors

### ISSUE THREAD

**Issue by Limeth** [sebcrozet/nalgebra#200]
It is common to create matrices with vectors as their columns or rows, and it would be great if there were dedicated constructor functions for this.

``` rust
fn new_with_columns(v1: Vector3<N>, v2: Vector3<N>, v3: Vector3<N>) -> Matrix3<N>;
fn new_with_rows(v1: Vector3<N>, v2: Vector3<N>, v3: Vector3<N>) -> Matrix3<N>;
```

**Comment by sebcrozet** [comment]
That would be useful. Though they should be named `.from_columns(...)` and `.from_rows(...)`. The `new_with_` prefix has been replaced in the version 0.9.0 because it is much less idiomatic than `from_`.

**Comment by sebcrozet** [comment]
Fixed by #218. See the constructors `::from_rows` and `::from_columns` [there](http://nalgebra.org/vectors_and_matrices/#matrix-construction).
